orphanage_APP/resources/views/graphs/donation-graph.blade.php

```diff
@@ -0,0 +1,36 @@
+<canvas id="myChart" width="400" height="50"></canvas>
+
+<script>
+    var ctx = document.getElementById('myChart');
```

Comment:
I think you should change the canvas id name into the appropriate graph name ie. Donations Graph